src/ontology/mondo-edit.obo

```diff
@@ -434085,6 +434039,93 @@ synonym: "hyperkalemic RTA" EXACT [https://www.niddk.nih.gov/health-information/
+property_value: seeAlso https://rarediseases.info.nih.gov/diseases/12592/oculopharyngodistal-myopathy xsd:anyURI {source="GARD:0012592"}
+created_by: http://orcid.org/0000-0001-5208-3432
+
+[Term]
+id: MONDO:0100167
+name: oculopharyngodistal myopathy 1
+def: "An autosomal dominant disorder characterized by adult-onset ptosis, external ophthalmoplegia, facial muscle weakness, distal limb muscle weakness and atrophy, and pharyngeal involvement, resulting in dysphagia and dysarthria. Skeletal muscle biopsy shows myopathic changes with rimmed vacuoles. There are variable manifestations of the disorder regarding muscle involvement and severity." [OMIM:164310]
+synonym: "faciooculolaryngopharyngeal myopathy with distal and respiratory involvement" RELATED [OMIM:164310]
+synonym: "OPDM1" EXACT [OMIM:164310]
+is_a: MONDO:0100166 {source="OMIM:164310"} ! oculopharyngodistal myopathy
+intersection_of: MONDO:0100166 ! oculopharyngodistal myopathy
+intersection_of: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/31708
+relationship: has_modifier HP:0000006 {source="OMIM:164310"} ! Autosomal dominant inheritance
+created_by: http://orcid.org/0000-0001-5208-3432
+
+[Term]
+id: MONDO:0100168
+name: Silver-Russell syndrome
+def: "Silver-Russell syndrome is characterized by growth retardation with antenatal onset, characteristic facies and limb asymmetry." [Orphanet:813]
+subset: ordo_disease {source="Orphanet:813"}
+synonym: "Russell Silver syndrome" RELATED [GARD:0004870]
+synonym: "Russell-Silver dwarfism" EXACT [NCIT:C85068]
+synonym: "Russell-Silver syndrome" RELATED [OMIMPS:180860]
+synonym: "Silver Russell dwarfism" EXACT [DOID:14681]
+synonym: "Silver Russell syndrome" RELATED [GARD:0004870]
+synonym: "Silver-Russell dwarfism" RELATED [OMIMPS:180860, Orphanet:813]
+synonym: "Silver-Russell syndrome" EXACT [MONDO:Lexical, OMIMPS:180860]
+synonym: "Silver-Russell syndrome" EXACT [MONDO:0008394]
+synonym: "SILVER-RUSSELL syndrome; SRS" RELATED [OMIMPS:180860]
+synonym: "SRS" RELATED [MONDO:Lexical, OMIMPS:180860]
+xref: DOID:14681 {source="MONDO:equivalentTo"}
+xref: GARD:0004870 {source="MONDO:equivalentTo", source="Orphanet-shared", source="OMIM-shared"}
+xref: ICD10:Q87.1 {source="DOID:14681", source="Orphanet:813", source="ORDO:813/ntbt", source="ORDO:813/inclusion"}
+xref: ICD9:759.89 {source="MONDO:relatedTo", source="i2s"}
+xref: MedDRA:10062282 {source="ORDO:813/e", source="Orphanet:813"}
+xref: MESH:D056730 {source="ORDO:813/e", source="DOID:14681", source="MONDO:equivalentTo", source="Orphanet:813", source="MONDO:ontobio"}
+xref: NCIT:C85068 {source="DOID:14681", source="MONDO:equivalentTo"}
+xref: OMIM:312780 {source="MONDO:superClassOf", source="Orphanet:813", source="ORDO:813/btnt"}
+xref: OMIM:616489 {source="MONDO:superClassOf", source="Orphanet:813"}
+xref: OMIMPS:180860 {source="ORDO:813/e", source="DOID:14681", source="MONDO:equivalentTo", source="Orphanet:813"}
```

Comment:
I don't think this is correct. The provenance for an OMIMPS did not come from ORDO